### ISSUE TITLE
Calculate Foodcoop credit balance

### ISSUE THREAD

**Issue by Tamriel** [foodcoops/foodsoft#382]
It would be nice, if in the finance overview screen the Foodcoop's credit balance would be displayed.
Each time an order with a price markup / Foodcoop margin is cleared, the earned money shall be added to the Foodcoop's account credit balance.
I think it should be a normal account, because then there is a history and one can remove money from the account.

**Comment by wvengen** [comment]
Hmm interesting. If you're interested in this, it would be useful to work this out, with questions like: What would you use this credit balance for? Would its balance to reflect the true current funds? Bank and or cash? How would the invoices come into play? Would the foodcoop account be able to order - and what does that mean? Does "clearing" here mean the same as "settling an order"? Etc.

I doubt whether it would be wise to add a full accounting system to Foodsoft -  perhaps connecting to something like [Odoo](https://github.com/odoo/odoo) for that would be more suitable. A simple solution that helps basic use-cases could be useful, I think.

**Comment by Tamriel** [comment]
In the 'Manage accounts' account screen I would like the sum of all account balances summed up, likes in this mockup:
![screenshot_20180131_175247](https://user-images.githubusercontent.com/5037153/35636102-69c6ff48-06b0-11e8-923e-4dc6598fe824.png)
We need this sum to check if it is equal to the money on the real bank account of the food coop.

**Comment by wvengen** [comment]
That shouldn't be too difficult. A suitable place might also be the finance overview. This may be an appropriate feature for a plugin.

**Comment by Tamriel** [comment]
The SQL command to calculate the balance is
```
SELECT SUM( account_balance )
FROM groups
WHERE deleted_at IS NULL
```

**Comment by yksflip** [comment]
This should be resolved by #1017
